texar/torch/modules/connectors/connectors.py

```diff
@@ -91,7 +81,7 @@ def _sum_output_size(output_size: OutputSize) -> int:
     if isinstance(flat_output_size[0], torch.Size):
         size_list = [0] * len(flat_output_size)
         for (i, shape) in enumerate(flat_output_size):
-            size_list[i] = np.prod([dim for dim in shape])
+            size_list[i] = np.prod(shape)
```

Comment:
I thought we've talked about this. We can write this in a much more Pythonic way:
```python
size_list = [np.prod(shape) for shape in flat_output_size]
```